src/stores/messages/MessagesStore.ts

```diff
@@ -197,6 +185,33 @@ export default class MessagesStore {
 		this.filterStore.setMessagesFilter(filter, sseFilters, isSoftFilterApplied);
 	};
 
+	private init = async (defaultState: MessagesStoreDefaultStateType) => {
+		if (!defaultState) {
+			return;
+		}
+		if (typeof defaultState === 'string') {
+			const message = await this.api.messages.getMessage(defaultState);
+			this.initFilterStore({
+				targetMessage: message,
+				streams: [message.sessionId],
+			});
+			return;
+		}
+		this.initFilterStore(defaultState);
```

Comment:
and this probably could be replaced by `this.filterStore = new MessagesFilterStore(this.searchStore, defaultState)`;

and then we can delete `initFilterStore`